code/game/turfs/closed/minerals.dm

```diff
@@ -36,8 +36,17 @@
 	M.Translate(-4, -4)
 	transform = M
 	icon = smooth_icon
-	var/static/list/behaviors = list(TOOL_MINING)
-	AddElement(/datum/element/bump_click, tool_behaviours = behaviors, allow_unarmed = TRUE)
+
+/turf/closed/mineral/Bumped(atom/movable/bumped_atom)
```

Comment:
that brings up another point actually
with this change, the element becomes unused. I think it could be used elsewhere, so I'm kinda tempted to leave it in, but it is currently dead code